davidpak/SmartNote
# Implemted summarize.py

## What type of PR is this? (check all applicable)

- [x ] Feature

## Description

## New Feature
* Implemented summarize.py, which takes in a youtube video link and outputs a short summary
* Also has a custom prompt feature that allows for custom prompts based on the video


## QA Instructions, Screenshots, Recordings

```cd``` into the ```server/scripts``` directory and run ```python summarize.py``` or press the run button on your respective IDE. For a custom prompt, update the ```query = _____``` variable on line 68 to your custom prompt. The output will appear in the output console.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x ] No, and this is why: Widescale testing will be expensive
